src/Integration/Service/DataModel/QualityProfile.cs

```diff
@@ -26,7 +26,7 @@ internal class QualityProfile
         [JsonProperty("language")]
         public string Language { get; set; }
 
-        [JsonProperty("default")]
+        [JsonProperty("isDefault")]
         public bool IsDefault { get; set; }
 
         [JsonIgnore] // Not set by JSON
```

Comment:
There's an extra empty line below this property. (Not added now, but should be removed)